src/stylesheets/article.css

```diff
@@ -6,3 +6,32 @@
 .article-preview:hover h3 {
   text-decoration: underline;
 }
+.article-preview .img {
+  height: 100%;
+  width: 100%;
+}
+.article-preview .img {
+  height: 100%;
+  width: 100%;
+  background-size: cover;
+  background-repeat: no-repeat;
+  -webkit-transition: all 0.5s;
+  -moz-transition: all 0.5s;
+  -o-transition: all 0.5s;
```

Comment:
you don't need these because we have `autoprefixer`
https://github.com/postcss/autoprefixer

Reply:
fix

Reply:
You can just add 
```css
transition: ease-in-out 300ms;
  ```
  not delete everything :)